deploy/assisted-installer-controller/assisted-installer-controller-role-binding.yaml

```diff
@@ -3,11 +3,11 @@ apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: assisted-installer-controller
   namespace: assisted-deployment
-subjects:
-  - kind: ServiceAccount
-    name: assisted-installer-controller
-    namespace: assisted-deployment
 roleRef:
   kind: ClusterRole
   name: assisted-installer-controller
   apiGroup: rbac.authorization.k8s.io
+subjects:
+  - kind: ServiceAccount
```

Comment:
is order of subjects vs roleRef significant?

Reply:
nope, but looks like in all project this in this order. wanted to be alligned

Reply:
ok

Reply:
nope. just to be same as operators